src/models/assistant-model.ts

```diff
@@ -37,7 +37,11 @@ const OpenAIType = types.custom({
  * @property {Object|null} thread - The assistant's thread used for the current chat, or `null` if no thread is active.
  * @property {ChatTranscriptModel} transcriptStore - The assistant's chat transcript store for recording and managing chat messages.
  * @property {boolean} isLoadingResponse - Flag indicating whether the assistant is currently processing a response.
- * @property {string[]} messageQueue - Queue of messages to be sent to the assistant. Used if CODAP generates notifications while assistant is processing a response.
+ * @property {string[]} codapNotificationQueue - Queue of messages to be sent to the assistant. Used if CODAP generates notifications while assistant is processing a response.
+ * @property {string[]} messageQueue - Queue of messages to be sent to the assistant. Used if user sends messages while assistant is processing a response.
+ * @property {boolean} showLoadingIndicator - Flag indicating whether to show a loading indicator while the assistant is processing a response.
```

Comment:
It may be helpful to explain why we have a `showLoadingIndicator` boolean in addition to `isLoadingResponse`.